eve/main.yml

```diff
@@ -21,12 +21,21 @@ models:
       name: Set docker tag latest property
       property: docker_tag_latest
       value: "%(prop:registry_url)s:latest-%(prop:product_version)s"
+  - SetPropertyFromCommand: &short_version
+      name: Set short version
+      property: short_version
+      command: echo "%(prop:product_version)s" | grep -o '^[0-9]*\.[0-9]*'
+  - SetProperty: &docker_tag_short_version
+      name: Set docker tag latest short version property
+      property: docker_tag_short_version
+      value: "%(prop:registry_url)s:latest-%(prop:short_version)s"
```

Comment:
Here I'm wondering if I should rather replace the `docker_tag_latest` property to the `short_version` having this 3 digit latest tag may not be useful for us and we may only need to keep the one with the major and minor version.

Reply:
@rahulreddy